src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -144,7 +144,7 @@ const convertTaskDetail = async (taskStatus, ports, userName, jobName, taskRoleN
     containerIp: taskStatus.attemptStatus.podHostIP,
     containerPorts,
     containerGpus,
-    containerLog: `http://${taskStatus.attemptStatus.podHostIP}:${process.env.LOG_MANAGER_PORT}/logs/${userName}/${jobName}/${taskRoleName}/${taskStatus.attemptStatus.podUID}/`,
+    containerLog: `https://${process.env.WEBPORTAL_URL}/log-manager/${taskStatus.attemptStatus.podHostIP}:${process.env.LOG_MANAGER_PORT}/logs/${userName}/${jobName}/${taskRoleName}/${taskStatus.attemptStatus.podUID}/`,
```

Comment:
there shouldn't be a change, you need to add a `sub_filter` section [here](https://github.com/microsoft/pai/blob/30987a3efa2c6c6c0cd051b76f5798f54d204ee9/src/pylon/deploy/pylon-config/location.conf.template#L274-L276)

Reply:
1. what if there's no https? http should work
2. what if there's no pylon? webportal should work at its port
3. webportal ip may be different from pylon ip